synapse/models/inet.py

```diff
@@ -596,8 +596,32 @@ def getModelDefs(self):
                         'doc': 'A group name string.'
                     }),
 
+                    ('inet:http:header', ('comp', {'fields': (('name', ('str', {'lower': True})), ('value', 'str'))}), {
+                        'doc': 'An HTTP protocol header key/value.',
+                    }),
+
+                    ('inet:http:param', ('comp', {'fields': (('name', ('str', {'lower': True})), ('value', 'str'))}), {
```

Comment:
I made `name` a str:lower on the form as it was on the prop but not the form whereas inet:http:header is lower on both ends.